src/main/java/org/intellij/sdk/codesync/files/AlertsFile.java

```diff
@@ -0,0 +1,159 @@
+package org.intellij.sdk.codesync.files;
+
+import org.intellij.sdk.codesync.CodeSyncLogger;
+import org.intellij.sdk.codesync.exceptions.FileLockedError;
+import org.intellij.sdk.codesync.exceptions.InvalidYmlFileError;
+import org.intellij.sdk.codesync.utils.CommonUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.Instant;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.intellij.sdk.codesync.Constants.*;
+
+/*
+This file is mainly for compatibility with VSCode plugin.
+at the moment we are not reading from this file in the intelliJ plugin. This may change in the future and we will
+update this message then.
+
+We can have the following data in alerts.yml
+    team_activity:
```

Comment:
fyi, VSCode currently only puts following data
```
team_activity:
  basit@codesync.com:
    checked_for: '2023-01-11'
    shown_at: 2023-01-11T14:31:33.086Z
```
It does not use user_activity and any other main key. Comment needs to be updated to reflect only required data, I will update in VSCode.